### PR TITLE
docs: updated folder structure for pages

### PR DESCRIPTION
**Pull request by webmandman**
Pages folder structure was duplicated 3 times to seperate subfolders. It was udpated to have all subfolder content under one pages folder.

Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/pages/en/folder-structure.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ Further down, the description of each folder indicates its purpose and if it is
 │  │  ├─ schema.mjs
 │  │  └─ server.mjs
 │  ├─ pages
+│  │  └─ api
+│  │  │  ├─ auth
+│  │  │  │  └─ [...nextauth].ts
+│  │  │  └─ trpc
```
Comment (juliusmarminge) [inline www/src/pages/en/folder-structure.md:26]: Wrong symbol

**Review by juliusmarminge — APPROVED** [review]
Thanks, didn't even notice that one 👍 just one little detail

File: www/src/pages/en/folder-structure.md (hunk 1 above)
Comment (webmandman) [reply www/src/pages/en/folder-structure.md:26]: ah yes I see it. The one before "api" line 23, should be sideways T.

Comment (webmandman) [reply www/src/pages/en/folder-structure.md:26]: oh and line 15 before trpc.

**Review by nexxeln — APPROVED** [review]

**Pull request merged**